### PR TITLE
Adds textarea input to blend module

### LINKED ISSUE #389: When the input is larger make it as textarea
In the blend module, the input is the blend function, according to which blending should be done. 
So the input area shows function in a single line, it would look good if we can use the text area or some other component for it, instead of input.

screenshot from 2018-10-06 01-34-24

### PR DESCRIPTION
**Pull request by Divy123**
Fixes #389 

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [x] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `npm test`
* [x] code is in uniquely-named feature branch and has no merge conflicts
* [x] PR is descriptively titled
* [x] ask `@publiclab/reviewers` and `@publiclab/is-reviewers` for help, in a comment below

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software

Thanks!

### CODE REVIEW

**Comment by Divy123** [comment]
@jywarren please review.

**Comment by Divy123** [comment]
@jywarren please have a look.

File: examples/lib/defaultHtmlStepUi.js [hunk 1]
```diff
@@ -81,27 +81,44 @@ function DefaultHtmlStepUi(_sequencer, options) {
...
-
-          if (inputDesc.type.toLowerCase() == "range") {
-            html +=
-              '"min="' +
-              inputDesc.min +
-              '"max="' +
-              inputDesc.max +
-              '"step="' +
-              inputDesc.step + '">' + '<span>' + paramVal + '</span>';
+          if (step.name === 'blend' && inputDesc.type === 'input') {
```
Comment (HarshKhandeparkar) [inline examples/lib/defaultHtmlStepUi.js]: Instead of checking for step name, how about make it an input type in the info.json file?

Comment (Divy123) [reply examples/lib/defaultHtmlStepUi.js]: Can you please explain a bit on this?
Thanks!!

Comment (HarshKhandeparkar) [reply examples/lib/defaultHtmlStepUi.js]: In the info.json file, we have all the inputs. Each input has its own type like `text`, `select` etc. Btw, there is not input type called `input` as it is mapped to a valid html input type in the [mapHtmltypes.js](https://github.com/publiclab/image-sequencer/blob/main/examples/lib/mapHtmltypes.js) file(added in a new pr). The only types accesible to the ui itself are `text`, `range`, `select`, `number`, anything else like `input` or `URL` is defaulted to `text`.

So I feel like we could add a new type called `textarea` in the info files and make subsequent changes to the mapHtmltypes.js file.

I hoe this explains everything. Thank you!

Comment (Divy123) [reply examples/lib/defaultHtmlStepUi.js]: Thank you so much for this.

Comment (HarshKhandeparkar) [reply examples/lib/defaultHtmlStepUi.js]: I think you are using an old verison. Please rebase. This is how it looks in the current main branch

https://github.com/publiclab/image-sequencer/blob/e1a113cde183a620eaa9bb0fd0c37520178eb84a/examples/lib/defaultHtmlStepUi.js#L79

**Comment by jywarren** [comment]
Hi, just checking if you've gotten stuck on this at all, or if I could help in any way? Thanks!

**Comment by Divy123** [comment]
actually right now I was exploring more into performance issues.
Will come to this very soon.

**Comment by HarshKhandeparkar** [comment]
This has become quite old. Closing for now. Please feel free to rebase and reopen :)

**Pull request closed without merging**